source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRHostConfig.mm

```diff
@@ -0,0 +1,142 @@
+//
+//  ACRHostConfig.mm
+//  ACRHostConfig.h
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "ACRHostConfig.h"
+
+using namespace AdaptiveCards;
+
+@implementation ACRHostConfig
+
+- (id) init {
+    self = [super init];
+
+    if(self)
+    { 
+        _m_hostConfig = std::make_shared<HostConfig>();
+    }
+
+    return self;
+}
+
+- (u_int32_t) retrieveIntFromHostConfigString: (NSString *) str{
```

Comment:
Thanks

---
In reply to: [122776553](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122776553) [](ancestors = 122776553)